fastlane/Fastfile

```diff
@@ -219,7 +219,15 @@ end
       ios_bump_version_beta()
     end
 
-    trigger_app_store_ci_build(branch_to_build: "release/#{ios_get_app_version()}")
+    # Wrap up
+    version = ios_get_app_version
+    branch = "release/#{version}"
+    removebranchprotection(repository: GHHELPER_REPO, branch: branch)
+    setfrozentag(repository: GHHELPER_REPO, milestone: version, freeze: false)
+    create_new_milestone(repository: GHHELPER_REPO)
+    close_milestone(repository: GHHELPER_REPO, milestone: version)
```

Comment:
See https://github.com/woocommerce/woocommerce-ios/blob/14dfd50e6e73f27a801c6b2aef59f9134f167c25/fastlane/Fastfile#L240-L245

As I mentioned in the PR description, the only thing I didn't add is the step to remove the incremental tags. I need to followup on that before making this PR ready for review.